Foundation/NSNumber.swift

```diff
@@ -198,14 +198,20 @@ extension NSNumber : ExpressibleByFloatLiteral, ExpressibleByIntegerLiteral, Exp
 
 }
 
+@_silgen_name("_CFNumberGetType2")
```

Comment:
ForFoundationOnly.h is used in both the CoreFoundation shipped on Darwin as well as swift-corlibs-foundation, whereas ForSwiftFoundationOnly.h is used for swift-corelibs-foundation exclusively.

Putting it in ForSwitFoundationOnly.h means that I can merge the closed source version and the open source version easier ;)